presto-main/src/test/java/com/facebook/presto/cost/TestExchangeStatsRule.java

```diff
@@ -153,13 +154,13 @@ public void testExchangeConfidence()
                         .addSource(pb.values())))
                 .withSourceStats(0, PlanNodeStatsEstimate.builder()
                         .setOutputRowCount(100)
-                        .setConfident(true)
+                        .setConfidence(FACT)
                         .build())
                 .withSourceStats(1, PlanNodeStatsEstimate.builder()
                         .setOutputRowCount(100)
-                        .setConfident(false)
+                        .setConfidence(FACT)
                         .build())
                 .check(check -> check
-                        .confident(false));
+                        .confident(FACT));
```

Comment:
this should be low.  You should make sure to check what the tests are doing as you update them so you are not changing the meaning. 
This test was checking that the confidence would be set appropritely for an exchange node based on its input.  It had confidence set to true for some of its sources and false for one of them, and it expected that the overall confidence for the exchange node would be set to false.  We want to maintain both that behavior and the test for it with the enum change, so this test should several inputs with fact or high confidence and one with LOW confidence and the node confidence should be considered "LOW"